opal/mca/btl/openib/btl_openib_xrc.c

```diff
@@ -43,22 +43,22 @@ OBJ_CLASS_INSTANCE(ib_address_t,
 bool mca_btl_openib_xrc_check_api()
 {
     void *lib = dlopen(NULL, RTLD_NOW); /* current program */
+    void *sym;
     if (!lib) {
         BTL_ERROR(("XRC error: could not find XRC API version"));
         return false;
     }
 
+    sym = dlsym(lib, OPAL_VERSIONNED_XRC_SYMBOL);
```

Comment:
You should absolutely not be using dlopen() / dlsym() directly.  Please use opal_dl_open() and friends (see `opal/mca/dl/base/base.h`).